packages/web-lib/hooks/useBalances.tsx

```diff
@@ -77,38 +99,64 @@ function	useBalances(props?: Types.TUseBalancesReq): Types.TUseBalancesRes {
 				const	token = element.token as string;
 				const	balanceOf = results[rIndex++] as BigNumber;
 				const	decimals = results[rIndex++] as number;
-				const	symbol = results[rIndex++] as string;
-				const	price = results[rIndex++] as BigNumber;
+				let		symbol = results[rIndex++] as string;
+				const	price = format.BN(props?.prices?.[toAddress(token)] || ethers.constants.Zero);
```

Comment:
Should we just rename this to `rawPrice` as it seems that's what it is in the line 112 https://github.com/yearn/web-lib/pull/113/files#diff-dacb45fe2f69ad47874e2f41e11c681669bb3296fefe01c42520c2211dc1b3d4R112

```suggestion
				const	rawPrice = format.BN(props?.prices?.[toAddress(token)] || ethers.constants.Zero);
```